content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,145 @@
+However, this brings us one step closer to releasing 0.4.15.
+
+Due to development resources and manpower constraints we cannot give an exact estimation when 0.4.15 will be released to the general public but we will eventually get there.
+We are sorry for the inconvenience!
+In the meantime you can try [nightly builds](/getbuilds) that are packed with the latest bugfixes, new features and whatnot but expect also new bugs and such!
+
+## Kernel Debugger overhaul
+Historically, ReactOS was built with the GCC compiler.
+To help with kernel development, a local kernel debugger (KDBG) was developed.
+Unfortunately, KDBG could only understand an in-house debug symbol format that was incompatible with Microsoft's PDB format.
+Because KDBG was a local debugger, it was able to interact with the user via a serial port connection or locally via screen display.
+Input could be entered via serial or a PS/2 keyboard.
+
+Later on, the possibility to compile ReactOS with MSVC was added, and with it, support for the WinDbg debugger.
+Kernel debugging with WinDbg works by having a debugging stub on the target which is capable of manipulating the state of the OS.
+The actual debugger is offloaded to a host computer.
+
+Hermès is updating KDBG with new features and bug fixes to make it easier for developers and testers alike to reliably debug ReactOS.
+Improvements by Hermès for KDBG include:
+* Isolating terminal and input-related functionalities from the rest of KDBG ([#5188](https://github.com/reactos/reactos/pull/5188));
+* Fixed an input buffer reading bug in `KdbpReadCommand()` and `KdReceivePacket()` routines ([#4914](https://github.com/reactos/reactos/pull/4914));
+* Reworked the boot-phase initialization of KD and KDBG ([#4892](https://github.com/reactos/reactos/pull/4892));
+* Moved the code handling of Dmesg buffer from screen provider to KDBG provider ([#5143](https://github.com/reactos/reactos/pull/5143));
+
+The goal of this work is to have a terminal version of KDCOM and separate KDBG into a dedicated KDCOM-like extension DLL, such as `kdrosdbg.dll` (see [#4600](https://github.com/reactos/reactos/pull/4600)).
+All of this effort is further coordinated with Timo Kreuzer.
+
+## UEFI Boot
+Thanks to the work of Timo Kreuzer (tkreuzer) our AMD64 build is now capable of booting on an increasing number of hardware.
+Thanks to Timo's work, we are working towards supporting modern devices with UEFI class 3 firmware.
+Work has been underway since the beginning of the year to transition FreeLoader, our default bootloader for ReactOS, to support UEFI on ARM32, ARM64, x86 and of course: AMD64.
+Hermès has been developing a system for passing the UEFI framebuffer information in a fashion that allows Windows XP to run on UEFI systems, while Justin Miller (The_DarkFire_) has been developing the UEFI freeloader build.
+
+On top of supporting booting ReactOS, other features are being built such as EFI chainloading and a bootmgfw compatible build of FreeLoader.
```

Comment:
```suggestion
On top of supporting booting ReactOS, other features are being built such as EFI chainloading and a bootmgfw-compatible build of FreeLoader.
```
What you call "bootmgfw-compatible build of FreeLoader" is a build of FreeLoader that can be started as a build application by the Windows Boot Manager ("bootmgfw"). I think this might be reworded in that way instead.